doc/tasks/README.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Aufgaben"
+---
+
+## Begriffsklärung
+
+Folgende Begriffe haben im Kontext der Aufgabenbeschreibung eine besondere Bedeutung und werden
+daher nochmal definiert:
+
+**Spielelement**
+
+Ein Spielelement ist ein "Objekt" mit bestimmten Eigenschaften und [Komponenten](../ecs/components/readme.md),
+welches im Dungeon platziert ist. Ein Beispiel hierfür wäre eine Truhe, also Objekt, welche eine Truhen-Textur
```

Comment:
Puh.... ich weiß nicht wie wichtig das hier ist, aber das aussehen hat eigentlich nix mit "Was bist du" zu tun. 
Du bist die Summe deiner Components, wie die Daten darin aussehen macht dich erstmal nicht aus.

Reply:
Die Truhentextur wird aber auch in einem Component gespeichert, oder?

Reply:
Das sind hier wieder zwei unterschiedliche Perspektiven, denke ich. Aus Sicht der Spielenden und Lehreneden ist die Textur wichtig, um die Erwartungshaltung für das Verhalten einer Entität zu beeinflussen. Für den internen systemischen Blick ist das natürlich nicht wichtig. Ich denke, wir sollten uns entscheiden, welche Perspektive wir hier einnehmen wollen. Da der Abstraktionsgrad für dieses Dokument ja relativ hoch ist und z.B.. auch der Begriff der "Spielmechanik", den wir in dem Dokument verwenden, sich eher an der Lehrenden-Perspektive orientiert, würde ich vorschlagen, dass wir das auch für den Rest des Dokuments so beibehalten. Das und das Ziel des Dokuments könnten wir allerdings nochmal im Text ergänzen. Deine Meinung dazu, @AMatutat ?

Reply:
@malt-r Der letzte Beitrag ist vom 16.03., und die Textstelle finde ich im aktuellen Dokument nicht mehr. Kann(st Du) diese Diskussion resolve(n|d werden)?